tutorial-code/tutorial-code.csproj

```diff
@@ -13,7 +13,7 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="JavascriptEngineSwitcher.ChakraCore" Version="2.4.18" />
+    <PackageReference Include="JavascriptEngineSwitcher.ChakraCore" Version="3.0.1" />
```

Comment:
We probably have to update every dependency in this tutorial code at the same time